src/tray.cpp

```diff
@@ -251,13 +251,13 @@ Tray::Tray(QObject *parent/* = nullptr*/)
     d->mIconTimer.setInterval(0);
 
     d->mActEnableNetwork = d->mContextMenu.addAction(Tray::tr("Enable Networking"));
-    d->mActEnableWifi = d->mContextMenu.addAction(Tray::tr("Enable Wi-fi"));
+    d->mActEnableWifi = d->mContextMenu.addAction(Tray::tr("Enable Wi-Fi"));
     d->mContextMenu.addSeparator();
     QAction * enable_notifications = d->mContextMenu.addAction(Tray::tr("Enable notifications"));
     d->mContextMenu.addSeparator();
     d->mActConnInfo = d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("dialog-information")), Tray::tr("Connection information"));
     d->mActDebugInfo = d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("dialog-information")), Tray::tr("Debug information"));
-    connect(d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("document-edit")), Tray::tr("Edit connections...")), &QAction::triggered
+    connect(d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("document-edit")), Tray::tr("Edit connections…")), &QAction::triggered
```

Comment:
I know, that we're living in 21st century, but I'm oldfashioned in this regard and don't want to have non-ASCII characters in the source code.